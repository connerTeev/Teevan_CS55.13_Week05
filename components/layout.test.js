import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Layout from './layout';

describe('Layout', () => {
  it('renders the nav link food.com', () => {
    render(<Layout />);

    const navLink = screen.getByRole('link', {
      name: /food.com/i,
    });

    expect(navLink).toMatchSnapshot();
  });
});
