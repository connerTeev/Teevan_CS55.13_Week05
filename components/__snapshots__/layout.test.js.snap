// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders the nav link food.com 1`] = `
<a
  href="/"
>
  Food.com
</a>
`;
